docs/app/Examples/addons/Radio/Types/index.js

```diff
@@ -23,7 +25,14 @@ const RadioTypesExamples = () => (
       title='Radio Group'
       description='Radios can be part of a group.'
       examplePath='addons/Radio/Types/RadioGroup'
-    />
+    >
+      <Message warning>
+        Radios in a group must be
```

Comment:
`&nbsp;` should probably be after the `be`? Otherwise it doesn't achieve anything different than a `{' ' }`.